app/models/question_builder.rb

```diff
@@ -2,6 +2,12 @@ class QuestionBuilder
 	def self.build(associate_consultant)
 		questions = {}
 
+		questions["Comments"] = Review::Question.new("Comments", " General Comments", ["project_worked_on", "role_description", "comments"],
+										"<p>\
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.